packages/website/src/SideNavigation.tsx

```diff
@@ -87,6 +87,7 @@ export const Navigation: FunctionComponent = () => {
             <CollapsibleSideSection title="Mantine">
                 <NavLink href="/mantine/Collection" label="Collection" />
```

Comment:
🤔 I don't see the collection link in the demo

Reply:
Demo updated, now you should. If not, probably a cache issue on your side